metron-analytics/metron-profiler/README.md

```diff
@@ -129,23 +129,28 @@ This section will describe the steps required to get your first "Hello, World!""
     }
     ```
 
-1. Ensure that test messages are being sent to the Profiler's input topic in Kafka.  The Profiler will consume messages from the `inputTopic` defined in the [Profiler's configuration](#configuring-the-profiler).  By default this is the `indexing` topic.
+1. Ensure that test messages are being sent to the Profiler's input topic in Kafka.  The Profiler will consume messages from the `inputTopic` defined in the Profiler's configuration (see [Configuring the Profiler](#configuring-the-profiler)).  By default this is the `indexing` topic.
 
 1. Check the HBase table to validate that the Profiler is writing the profile.  Remember that the Profiler is flushing the profile every 15 minutes.  You will need to wait at least this long to start seeing profile data in HBase.
     ```
     $ /usr/hdp/current/hbase-client/bin/hbase shell
     hbase(main):001:0> count 'profiler'
     ```
 
-1. Use the Profiler Client to read the profile data.  The below example `PROFILE_GET` command will read data written by the sample profile given above, if 10.0.0.1 is one of the input values for `ip_src_addr`.
-More information on configuring and using the client can be found [here](../metron-profiler-client).
-It is assumed that the `PROFILE_GET` client is correctly configured before using it.
+1. Use the [Profiler Client]((../metron-profiler-client)) to read the profile data.  The following `PROFILE_GET` command will read the data written by the `hello-world` profile. This assumes that `10.0.0.1` is one of the values for `ip_src_addr` contained within the telemetry consumed by the Profiler.
+
     ```
     $ bin/stellar -z node1:2181
     [Stellar]>>> PROFILE_GET( "hello-world", "10.0.0.1", PROFILE_FIXED(30, "MINUTES"))
     [451, 448]
     ```
 
+    This result indicates that over the past 30 minutes, the Profiler stored two values.  In the first 15 minute period, the IP `10.0.0.1` was seen in 451 telemetry messages.  In the second 15 minute period, the same IP was seen in 448 telemetry messages.
```

Comment:
I think it would be more correct to say, `over the past 30 minutes, the "hello-world" Profiler stored two values related to the ip_src_addr "10.0.0.1".` 

(because it may have stored additional values for other ip_src_addr addresses that aren't mentioned here)